docs/components/tables/index.html

```diff
@@ -89,19 +133,23 @@ <h2 id="basic-table">Basic Table</h2>
   </section>
 
   <section><!-- Selection Table -->
-    <h2 id="selection-table">Selection Table</h2>
-    <p>
-      Use the <code>.hxControl</code> CSS class on table cells that contain
-      a checkbox.
-    </p>
+    <header>
+      <h2 id="selection-table">Selection Table</h2>
+      <p>
+        Use the <code>.hxControl</code> CSS class on table cells that contain
+        a checkbox.
+      </p>
+    </header>
+
     <div class="example">
       <div>
         <table class="hxTable">
           <caption>Most popular Twitter accounts</caption>
           <thead>
             <tr>
               <th class="hxControl">
-                <hx-checkbox></hx-checkbox>
```

Comment:
Changed to vanilla checkbox to reduce confusion as we figure out a label-less solution.